### ISSUE TITLE
test_bug_3270_at_path fails for Viktor on Python 3.11

### ISSUE THREAD

**Issue by edreamleo** [leo-editor/leo-editor#3369]
Closed via PR #3373.  See also: #3270.

```console
FAIL: test_bug_3270_at_path (unittests.core.test_leoAtFile.TestAtFile.test_bug_3270_at_path)
    ----------------------------------------------------------------------
    Traceback (most recent call last):
      File "/home/user/PyVE/GitHub/Leo/leo-editor/leo/unittests/core/test_leoAtFile.py", line 102, in test_bug_3270_at_path
        self.assertEqual(path, expected)
    AssertionError: '/home/user/PyVE/GitHub/Leo/leo-editor/leo/core/c:/temp/leo/at_file_test.py' != 'c:/temp/leo/at_file_test.py'
    - /home/user/PyVE/GitHub/Leo/leo-editor/leo/core/c:/temp/leo/at_file_test.py
    + c:/temp/leo/at_file_test.py
```

**Comment by boltex** [comment]
Might be that python 3.11 now considers a path starting with "C:/" on a linux / posix system NOT to be an absolute path, therefore prefixing with it's home path. The specific line that does this is 

`path = os.path.abspath(path)`

in finalize_join function of leoGlobals.py.

So happens that i'm trying to emulate the proper behavior of 'test_bug_3270_at_path' in the unit tests of leojs In leojs version of leoGlobals, I use the 'equivalent' path library from nodejs: also called 'path'. which contains method very similar to the python 'os.path' library, but with some small subtle differences that i'm trying to emulate properly.

**Comment by boltex** [comment]
a test to be done would be to run this (in both python above and below 3.11 to see the difference)

`print( os.path.abspath( 'c:/temp/leo/at_file_test.py' ) )`

**Comment by boltex** [comment]
a test to be done would be to run this (in both python above and below 3.11 to see the difference)

`print( os.path.abspath( 'c:/temp/leo/at_file_test.py' ) )`

**Comment by edreamleo** [comment]
Félix and I chatted about this issue on zoom. Our conclusion: this unit test can safely be deleted:
- The test doesn't actually test the original issue.
- The `c:/temp` syntax should not be used on linux.

I'll close this issue after deleting the test.

**Comment by boltex** [comment]
After doing more tests, I realize now that I wrongfully thought that this test existed since a long time, and that a newer python may have changed os.path.abspath.

 It was in fact always the same. This '**test_bug_3270_at_path**' test was simply pretty new, and not reported as giving an error when run under linux until Victor tried it.

(I've since found a way to reliably test under linux, and I'll make sure to run unit tests in linux it when Edward pulls something new in devel to help along with testing :) )

Anyways, as we discussed, removing this test makes sense.
